GitCommands/FullPathResolver.cs

```diff
@@ -56,8 +56,8 @@ public string Resolve(string path)
                 basePath += Path.DirectorySeparatorChar;
             }
 
-            var uri = new Uri(new Uri(basePath), path);
-            return uri.LocalPath;
+            var uri = Path.Combine(basePath, path);
```

Comment:
❗️  This will won't work as you expect it - `path` can be anything (e.g. `..\..\`):
![image](https://user-images.githubusercontent.com/4403806/75640630-d1e5e180-5c89-11ea-88e6-d76ef3a6123c.png)

We must call `Path.GetFullPath()` to ensure the result is correct:
![image](https://user-images.githubusercontent.com/4403806/75640676-f641be00-5c89-11ea-88fe-f94d0e0732d1.png)


💭 We should probably enhance the docs to state that while this method resolves paths it doesn't check or ensure the path exists.

Reply:
The results are what I expect
But documenting is OK I guess